recipes/unittest-xml-reporting/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "unittest-xml-reporting" %}
+{% set version = "3.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e09b8ae70cce9904cdd331f53bf929150962869a5324ab7ff3dd6c8b87e01f7d
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py27]
```

Comment:
I think this can just be `noarch: python` and then change  `python` under `host` and `run` to `python >=3.5` and remove the `skip`.